README.md

```diff
@@ -51,6 +50,12 @@ yarn add @rocketseat/unform
 
 ## Table of contents
 
+- [Overview](#overview)
```

Comment:
I find it is a bit odd to link a item that is above the table of contents.

Perhaps we should either move them down, or remove this link (along with all the other links that go to items above the ToC).

Dunno, just an idea. What do you guys think @italomlp @diego3g @pellizzetti ?

This is just a thought, not necessarily a required change.

Reply:
Yeah, linking those items makes sense but we should definitely move our ToC up.

Reply:
Yeah, I agree with both of you. I'll change it ASAP.

Reply:
I made the changes, bringing the ToC to right after the `Overview` section. This seems good for me, as I look in other nice repositories and most of them have the Overview or Resume before the ToC. Also, I removed the `Overview` link, because of the same reason as above.